src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.tizen.cs

```diff
@@ -90,11 +90,7 @@ void MouseUp()
 
 			isDrawing = false;
 
-			if (Element.Points.Count > 0)
-			{
-				if (Element.DrawingCompletedCommand?.CanExecute(null) ?? false)
-					Element.DrawingCompletedCommand.Execute(Element.Points);
-			}
+			Element.OnDrawingCompleted();
 
 			if (Element.ClearOnFinish)
```

Comment:
maybe it should be removed

Reply:
What should be removed?

Reply:
What should be removed?

Reply:
The `Clear` function does some tidy up logic in the renderer so I think it needs to stay.

Reply:
for example on Android, you removed this method, while it stays here. 
I understand that you wanted to removed duplicated code and move it to the shared OnDrawingCompleted, but because of such additional logic in Clear method, I think it's better to keep it in all renderers.
I implemented the same in DrawingView MultiLine PR.